ares/n64/aleck64/io.cpp

```diff
@@ -0,0 +1,59 @@
+auto Aleck64::readWord(u32 address, Thread& thread) -> u32 {
+  if(address <= 0xc07f'ffff) {
+    return sdram.read<Word>(address);
+  }
+
+  controls.poll();
+  if(address <= 0xc080'0fff) {
+    switch (address & 0xffff'fffc) {
+      case 0xc080'0000: return readPort1();
+      case 0xc080'0004: return readPort2();
+      case 0xc080'0008: return readPort3();
+    }
```

Comment:
Communication to the Z85233 RS-485 device that manages VS mode lives at `0xc080'0100`
```suggestion
      case 0xc080'0008: return readPort3();
      case 0xc080'0100: return readPort4(); // can just be a stub for now, game is happy without it
    }
```